examples/upload-photo-from-web.example.ts

```diff
@@ -0,0 +1,27 @@
+/* tslint:disable:no-console */
+import 'dotenv/config';
+import { IgApiClient } from '../src';
+const axios = require('axios'); // axios is used for this example, but you can use any library that you want
```

Comment:
We already have `request` library in `package.json` dependencies, please, use it instead of `axios`